Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -425,7 +439,7 @@ Object.defineProperties(ModelExperimental.prototype, {
       if (!Color.equals(this._color, value)) {
         this.resetDrawCommands();
       }
-      this._color = value;
+      this._color = Color.clone(value);
```

Comment:
To avoid an allocation...

```suggestion
      this._color = Color.clone(value, this_color);
```